docs/source/global_settings.rst

```diff
@@ -20,6 +20,14 @@ and the company ID.
 
 If the API endpoint ever changes it can be updated in this file.
 
-The company ID can be found on the `GameBench Web Dashboard <https://web.gamebench.net>`__.
-Once signed in go to the 'My Account' section.  The company id is the alpha-numeric string
-in the URL.
+Currently the company ID must be manually queried for.  Here are steps on retrieving the ID:
```

Comment:
I think this would flow better written as:
"Currently, the company ID must be manually retrieved. Here are the steps to do that:"

Let me know what you think!

Reply:
Thanks for the feedback!  Implemented some changes to the wording.

Reply:
Much better, thanks!